backstop_data/engine_scripts/puppet/contributions/find-contributions-delete.js

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
also the name of the file doesn't make sense, what does it mean "find contributions delete"? It's very confusing

Reply:
i see that this is a pattern with all file names in this PR, please choose more descriptive names

Reply:
@swastikpareek you didn't address this